Makefile.ibm

```diff
@@ -19,8 +19,9 @@ DOCKER_REGISTRY_DOCKERHUB := registry.hub.docker.com
 DOCKER_USER_DOCKERHUB := $(DOCKER_HUB_USERNAME)
 DOCKER_PASS_DOCKERHUB := $(DOCKER_HUB_API_KEY)
 
-DOCKER_IMAGES := detect-secrets detect-secrets-hook
-DOCKER_IMAGES_TAGGED := detect-secrets detect-secrets:redhat-ubi detect-secrets:redhat-ubi-custom detect-secrets-hook
+DOCKER_IMAGES_TO_TAG := detect-secrets detect-secrets-hook
+DOCKER_IMAGES_TO_TRIVY := detect-secrets detect-secrets-hook detect-secrets:redhat-ubi detect-secrets:redhat-ubi-custom
```

Comment:
Small suggestion: would you be open to renaming `DOCKER_IMAGES_TO_TRIVY` to `DOCKER_IMAGES_TO_SCAN`?. In `docker-quality-images`, the `trivy image` command is run to scan each Docker image for vulnerabilities.

Reply:
done